.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   nest_tests:
     docker:
-      - image: circleci/node:10.18-browsers
+      - image: cimg/node:16.15-browsers
```

Comment:
Are you sure that we need this? I thought circleci files like these are leftovers from when Labs used to use it but doesn't anymore, but you would have more context into the history of that than me.

Reply:
Hmm interesting. Do you happen to know how where I might find login credentials for circleci? I asked around about it but didn't get any answers.

Reply:
Hmm not sure which GH account that would be but I'm fine with this change in the mean time.